src/PostgREST/Error.hs

```diff
@@ -92,7 +92,7 @@ instance JSON.ToJSON ApiRequestError where
     "hint"    .= JSON.Null]
   toJSON InvalidRange = JSON.object [
     "code"    .= ApiRequestErrorCode03,
-    "message" .= ("HTTP Range error" :: Text),
+    "message" .= ("Requested range not satisfiable" :: Text),
```

Comment:
Could we further clarify this with a message like `an offset with 1000 was requested but there are only 500 rows`?

Reply:
I added this to the "details". It also kinda forced me to add details for other 416 errors like negative limits and where the lower boundary is greater than the upper one in the Range header.